language/compiler/src/main.rs

```diff
@@ -42,6 +42,9 @@ struct Args {
     /// Instead of compiling the source, emit a dependency list of the compiled source
     #[structopt(short = "-l", long = "list_dependencies")]
     pub list_dependencies: bool,
+    /// Path to the list of modules that we want to link with
```

Comment:
What format is this list stored in?

Reply:
It will be a vector of blob. For the current code, it was using serde_json.